### PR TITLE
Publish artifacts from forks

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

**Comment by sebastienros** [comment]
The previous PR was wrongly targeting master

**Comment by natemcmaster** [comment]
Requesting @Eilon to take a look. This means artifacts are uploaded for all public PRs.

**Comment by Eilon** [comment]
Hmm yes that is a no-go. We cannot publish artifacts for arbitrary PRs.

File: .azure/templates/phases/default-build.yml [hunk 1]
```diff
@@ -94,7 +94,6 @@ phases:
   - ${{ if eq(parameters.artifacts.publish, 'true') }}:
     - task: PublishBuildArtifacts@1
       displayName: Upload artifacts
-      condition: eq(variables['system.pullrequest.isfork'], false)
```
Comment (natemcmaster) [inline .azure/templates/phases/default-build.yml]: Suggest keeping this as:
```
condition: always()
```
This ensures artifacts are uploaded even if the tests failed. This is useful for debugging test failures.

**Comment by sebastienros** [comment]
> Hmm yes that is a no-go. We cannot publish artifacts for arbitrary PRs.

@davidfowl so benchmark bot will be just for PR from local branches

**Comment by sebastienros** [comment]
@Eilon would it be ok, if it's possible and if this is the concern, to publish them only in the internal CI?

**Comment by Eilon** [comment]
Why would we want artifacts from random forks?

**Pull request closed without merging**